Dockerfile.arm

```diff
@@ -21,7 +25,9 @@ RUN find . -type d -name obj | xargs -r rm -r
 RUN dotnet publish Jellyfin.Server --configuration Release --output="/jellyfin" --self-contained --runtime linux-arm "-p:GenerateDocumentationFile=false;DebugSymbols=false;DebugType=none"
 
 
-FROM debian:buster-slim
+FROM multiarch/qemu-user-static:x86_64-arm as qemu
+FROM arm32v7/debian:buster-slim
+COPY --from=qemu /usr/bin/qemu-arm-static /usr/bin
 RUN apt-get update \
  && apt-get install --no-install-recommends --no-install-suggests -y ffmpeg \
  libssl-dev ca-certificates \
```

Comment:
```suggestion
FROM multiarch/qemu-user-static:x86_64-arm as qemu
FROM lsiobase/ubuntu:arm32v7-bionic
COPY --from=qemu /usr/bin/qemu-arm-static /usr/bin

ARG DEBIAN_FRONTEND="noninteractive"
ENV NVIDIA_DRIVER_CAPABILITIES="compute,video,utility"
ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn

RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates gnupg curl && \
 curl -s https://repo.jellyfin.org/ubuntu/jellyfin_team.gpg.key | apt-key add - && \
 curl -s https://keyserver.ubuntu.com/pks/lookup?op=get\&search=0x6587ffd6536b8826e88a62547876ae518cbcf2f2 | apt-key add - && \
 echo 'deb [arch=armhf] https://repo.jellyfin.org/ubuntu bionic main' > /etc/apt/sources.list.d/jellyfin.list && \
 echo "deb http://ppa.launchpad.net/ubuntu-raspi2/ppa/ubuntu bionic main">> /etc/apt/sources.list.d/raspbins.list && \
  apt-get install --no-install-recommends --no-install-suggests -y \
 	jellyfin-ffmpeg \
	libssl-1.0.0 \
	libfontconfig1 \
	libfreetype6 \
	libomxil-bellagio0 \
	libomxil-bellagio-bin \
	libraspberrypi0 \
```

Reply:
Don't hate me lol. Feels like hackery, and it should definitely be cleaned up

Reply:
If this works, I'm good with it. Can't seem to apply the suggestions though. Open a separate PR.